### PR TITLE
Add sample/test for blob_detector

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: jsk_perception/sample/sample_blob_detector.launch [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<launch>
+
+  <arg name="gui" default="true" />
+
+  <node name="image_publisher"
+        pkg="jsk_perception" type="image_publisher.py">
+    <param name="file_name" value="$(find jsk_perception)/sample/image/mask0001.jpg" />
```
Comment (k-okada) [inline jsk_perception/sample/sample_blob_detector.launch]: how did you make this image? computer generated? or copied from somewhere? I prefer to use more descriptive name

**Comment by wkentaro** [comment]
Fixed.

**Comment by wkentaro** [comment]
Test fails.
```
jsk_pcl_ros_utils/test_results/jsk_pcl_ros_utils/MISSING-rostest-test_cluster_point_indices_to_point_indices.xml: 1 tests, 0 errors, 1 failures
```

**Comment by k-okada** [comment]
Related? https://github.com/jsk-ros-pkg/jsk_recognition/pull/1848

**Comment by wkentaro** [comment]
I don't think so. The test failed because of the failed making directory of test_result directory.

**Pull request merged**